Sources/CCFilterNode/CCMapFilterNode.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+extension CCMapFilterNode {
+    enum Key: String, CaseIterable {
+        case lat, lon, radius, geoLocationName
```

Comment:
Perhaps `latitude` and `longitude` would be more readable?

Reply:
Agree, better to use `latitude` and `longitude`. And newlines after each case (a general recommendation, not just for this enum)

Reply:
Agreed!